.github/workflows/validate-stacks.yaml

```diff
@@ -90,6 +91,17 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: 0
+      
+      - name: Free Disk Space
+        uses: jlumbroso/free-disk-space@main
```

Comment:
Would it be possible to pin this to a revision hash? Also, use a revision hash of the latest version `v1.3.1` if it has what we need?